torch/_dynamo/utils.py

```diff
@@ -143,6 +147,51 @@
 timer_counter = itertools.count()
 
 
+# Abstraction on top of counters.
+class ReInplaceTrigger(enum.Enum):
```

Comment:
can you message me a query on the scuba table sometime?

Reply:
yeh i will share the query once i and it and collect the new data